README.md

```diff
@@ -291,11 +291,23 @@ npm run test:schemas
    [schemas](./packages/traceability-schemas/schemas) folder.
 2. Add synthetic data generation for it to the
    [generators](./packages/traceability-schemas/src/generators).
-3. OPTIONAL -- Add your new schema name to the file [includingevidence.json](./packages/traceability-schemas/src/data/vc/includingevidence.json) if you want to include the ```evidence``` property in your verifiable credential test fixture.
-4. OPTIONAL -- Add your new schema name to the file [includincredentialstatus.json](./packages/traceability-schemas/src/data/vc/includingcredentialstatus.json) if you want to include the ```credentialStatus``` property in your verifiable credential test fixture.   
-5. Run the build using: `npm run build:all`
-6. Fix any errors found.  Repeat as needed.
-7. Review the latest spec changes by serving docs: `npx serve ./docs`.
+3. OPTIONAL — If you want to include the ```evidence``` property in your
+   verifiable credential test fixture, add your new schema name to the
+   [includingevidence.json](./packages/traceability-schemas/src/data/vc/includingevidence.json)
+   file.
+5. OPTIONAL — If you want to include the ```credentialStatus``` property in
```

Comment:
I think you missed 4.

Reply:
It's MD. I could use `1.` throughout or any assortment of numbers in the MD source; it'll go to clients as an HTML ordered list with ascending numerals.